src/pyscanner.l

```diff
@@ -261,6 +261,12 @@ STARTDOCSYMS      "##"
 			  searchFoundDef(yyscanner);
 			  BEGIN( FunctionDec );
       			}
+    ^{B}"async"{BB}"def"{BB} { // start of an async function/method definition with indent
+      			  DBG_CTX((stderr,"Found async def at %d\n",yyextra->yyLineNr));
+      			  yyextra->indent=computeIndent(yytext);
+			  searchFoundDef(yyscanner);
+			  BEGIN( FunctionDec );
+      			}
```

Comment:
- Can you give a reference to `async` in the python documentation (e.g. a pep)?
- You created a new rule analogous to `^{B}"def"{BB}` , why isn't there a rule analogous to `    "def"{BB}`?

Reply:
@albert-github The PEP can be found here https://www.python.org/dev/peps/pep-0492/ (feature seems to be available since Python 3.5)

Reply:
Thanks for the pep.
Shouldn't the word `await` be handled as well? I didn't test it, might not cause any problems in the pyscanner.l, but in the pycode.l it won't be marked as a keyword.

Reply:
@albert-github Because I don't know why "def"{BB} is also defined although ^{B}"def"{BB} is defined. BTW, you look right I should add a patch for pycode.l. I'll make another patchset and push it in a few days.

Reply:
Probably the `"def"{BB}` is also defined for cases where there are multiple statements on one line (not sure whether this is anyway possible though). There must have been a reason for it as the split has already been made in 2005.
I think it is wise to add the rule with `async` for this case as well.

Reply:
When looking at the documentation:

![scanner1](https://user-images.githubusercontent.com/5223533/104904549-90355680-5981-11eb-8c0d-4e1816f2c689.png)

I don't see any reference to the `async` status, shouldn't this me mentioned in the blue part as well?